curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-026.md

```diff
@@ -1,28 +1,78 @@
 ---
-id: 6193ee19d10be144d6bc9c46
+id: 61fd9ad665a4a282c8106be3
 title: Step 26
 challengeType: 0
 dashedName: step-26
 ---
 
 # --description--
 
-Now that you have all of the financial data in place, you need to style your balance sheet so it looks visually appealing.
-
-Start by creating a `body` selector, and give it a `text-align` property set to `center`.
+Within your `thead`, add a `td` and three `th` elements. Within each of the `th` elements, add a `span` element with the `class` set to `sr-only` and the following text, in order: `2019`, `2020`, and `2021`.
```

Comment:
Not sure what to do with this one - I was a little confused. The description doesn't have you add a `tr` like the rest of the tables. I looked at the tests and it looks like it's being tested for. Turns out, the `tr` is rendered automatically if you add a `td` or something else in the `thead`.

We could add a step in between 25 and 26 that has users add the `tr` - or tell them to add the `tr` on this step. Or we could just leave it really. I'll let you decide @nhcarrigan.

Reply:
I'll go ahead and shove it in this step.